test/machine.test.ts

```diff
@@ -218,7 +218,7 @@ describe('machine', () => {
     });
 
     it('should resolve the state value', () => {
-      const tempState = State.from('foo');
+      const tempState = State.from<any>('foo');
```

Comment:
this is needed now as unparameterized generics are now unknown by default (instead of any)